support-metrics-client/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>commons-validator</groupId>
       <artifactId>commons-validator</artifactId>
-      <version>1.4.1</version>
```

Comment:
can we remove the version from this pom and instead rely on `dependencyManagement` from parent poms like the common pom?

Reply:
I'm not sure what you mean, I don't see `dependencyManagement` used in any of the support-metrics repos.

Reply:
Could we make a Jira for this and address it in a follow up PR? We're trying to get a build out to a client with a new version of commons-collection, the transitive dependency.

Reply:
Unless it's an easy fix of course.

Reply:
Ok thank you, I'll file a Jira for this.

Reply:
@gardnervickers The idea is to define the dependency in the `common` pom and then here you don't have to mention the version at all. And yes, we can do that in a follow-up, it's pretty trivial.

Reply:
Ok great, I made CP-607 to track this.

Reply:
In the mean time if either of you have a moment to approve this and unblock the merge, that would be appreciated 👍